test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -99,11 +149,66 @@ func makeBasePodWithAntiAffinity(podLabels, affinityLabels map[string]string) *v
 	return basePod
 }
 
+// makeBasePodWithPodAffinity creates a Pod object to be used as a template.
+// The Pod has a PodAntiAffinity requirement against pods with the given labels.
+func makeBasePodWithPodAffinity(podLabels, affinityZoneLabels map[string]string) *v1.Pod {
+	basePod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "affinity-pod-",
+			Labels:       podLabels,
+		},
+		Spec: testutils.MakePodSpec(),
+	}
+	basePod.Spec.Affinity = &v1.Affinity{
+		PodAntiAffinity: &v1.PodAntiAffinity{
```

Comment:
Wait ... `PodAntiAffinity` in `makeBasePodWithPodAffinity` test?

```suggestion
		PodAffinity: &v1.PodAffinity{
```

Also, ditto the comment of this function :-)

Reply:
Apologize... A copy/paste error..

Fixed.